docs/aerodromes/Amberley.md

```diff
@@ -4,7 +4,78 @@
 
 --8<-- "includes/abbreviations.md"
 
-Reserved.
+## Positions
 
+| Name               | Callsign       | Frequency        | Login Identifier              |
+| ------------------ | -------------- | ---------------- | --------------------------------------|
+| Amberley ADC    | Amberley Tower  | 118.300         | AM_TWR        |
+| Amberley SMC    | Amberley Ground  | 129.350      | AM_GND        |
+| Amberley ACD    | Amberley Delivery  | 134.600         | AM_DEL       |
+| Amberley ATIS    |   | 123.300         | YAMB_ATIS       |
+
+
+
+## Airspace
+
+AM ADC owns the Class C airspace in the AM CTR within 10nm of the YAMB ARP from `SFC` to `A015`. This airspace is primarily used for military circuits and initial and pitch approaches.
+
+
+## YAMB Aerodrome
+
+### SID Selection
+Aircraft planned via **BN**, **JEDDA**, **MESED**, **BOBOP**, **TATEN**, shall be assigned the **Procedural SID** that terminates at the appropriate waypoint.
+Aircraft who are not planned via those points or who are negative RNAV may be assigned a RADAR SID or a visual departure. 
+
+#### Coded Departures (Fast Jets)
+Visual Departures are commonly requested which are often in the form of a coded departure (e.g. BYRON7 departure), which can be found in the AD2 Sups Page for YAMB. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east of Brisbane avoiding civilian traffic inbound to Brisbane. 
+
```

Comment:
Include an example phraseology of an airways clearance.

"CRNG21, Cleared to YAMB via MOSSI, Flight Planned Route. Runway 15, Northern 1 Departure. Climb to F190. Squawk 3601, Depart 126.20"

Need to emphasise that they are cleared to their destination (so as not to impose a clearance limit without INL knowing), and that the level is restricted as well.